hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ListIterator.java

```diff
@@ -144,7 +149,7 @@ private void getNextKey() throws IOException {
       while (tableIterator.hasNext() && currentKey == null) {
         Table.KeyValue<String, Value> entry = tableIterator.next();
         String entryKey = entry.getKey();
-        if (!KeyManagerImpl.isKeyInCache(entryKey, table)) {
+        if (!isKeyExistsInCache.test(entryKey)) {
```

Comment:
seems some problem in implementation of DBTableIterator,
- hasNext() moves to next key by calling getNextKey()
- next() also call hasNext() implicitly calling getNextKey() and moves to next key
So while iterator, element can move twice skipping entries